src/internal/clusterstate/v2.7.0/coredb.go

```diff
@@ -0,0 +1,92 @@
+package v2_7_0
+
+import (
+	"context"
+	"time"
+
+	proto "github.com/gogo/protobuf/proto"
+
+	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/pachsql"
+	"github.com/pachyderm/pachyderm/v2/src/pfs"
+)
+
+// CollectionRecord is a record in a collections table.
+type CollectionRecord struct {
+	Key       string    `db:"key"`
+	Proto     []byte    `db:"proto"`
+	CreatedAt time.Time `db:"createdat"`
+	UpdatedAt time.Time `db:"updatedat"`
+}
+
+func createCoreSchema(ctx context.Context, tx *pachsql.Tx) error {
```

Comment:
Are the other tables going to be added to this function over time? It seems like the table creation should be in the apply function that is performing the migration, and the future tables should just be a part of additional apply functions.